src/test/java/de/zalando/ep/zalenium/container/DockerContainerClientTest.java

```diff
@@ -0,0 +1,57 @@
+package de.zalando.ep.zalenium.container;
+
+import com.spotify.docker.client.DefaultDockerClient;
+import com.spotify.docker.client.DockerClient;
+import com.spotify.docker.client.messages.ContainerConfig;
+import de.zalando.ep.zalenium.util.DockerContainerMock;
+import java.util.HashMap;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+
+public class DockerContainerClientTest {
+
+  private DockerClient dockerClient;
+  private String zaleniumContainerId;
+  private DockerContainerClient containerClient;
+
+  @BeforeTest
+  public void prepare() throws Exception {
+    dockerClient = new DefaultDockerClient("unix:///var/run/docker.sock");
```

Comment:
Can you please use mocks instead of the real docker daemon? This would slow down a lot the unit test execution. There is a mock in the TestUtils class (which doesn't look so nice and needs rework), maybe you can reuse it to add the test.